### DISCUSSION TITLE
Schimid factor tutorial question

### DISCUSSION THREAD

**Discussion by hsheng94** [mtex-toolbox/mtex#1342] [Ask Anything]
I am reading the tutorial on page https://mtex-toolbox.github.io/SchmidtFactor.html
Here is a quote from the example. There use 2 different set of codes for computing the Schmid factor. Intuitively, the result should be the same. However, the result is apparently not. 
![image](https://user-images.githubusercontent.com/26800081/153093509-7346e75d-9818-4a17-b237-9eb3e73fa2f6.png)
I checked the source code in github and I believe the code should execute this line. This should give the same result as the dot product methods and output 0.4286. So I am so confused about how the 0.17 is computed from the code sS.SchimidFactor（r）
![image](https://user-images.githubusercontent.com/26800081/153093613-929540ad-d4e4-4464-97a7-276dc384b432.png)

**Comment by kilir** [comment]
Hi,
actually SchmidFactor() normalizes the slip system (as well as might optionally consider the CRSS).
```
tau = dot(d.normalize,r,'noSymmetry') * dot(n.normalize,r,'noSymmetry')
```
should give the same result. It's probably a good idea to put this in the demo.

Cheers,
Rüdiger
